### PR TITLE
Added callout for verify v2 WABA + email

### PR DESCRIPTION
**Pull request by SecondeJK**
# Description

# New 

* Added callout for v2 to notify customers about WABA + custom from email

# Checklist

- [x] version number incremented (in the `info` section of the spec)

### CODE REVIEW

File: definitions/verify.v2.yml [hunk 1]
```diff
@@ -12,18 +12,26 @@ servers:
...
-      description: Start verifying that we can reach this user given a set of contact details
-      tags:
-        - Requests
+      description: >-
+        Start verifying that we can reach this user given a set of contact details
+
+        <div class="Vlt-callout Vlt-callout--critical">
+          <div class="Vlt-callout__content">
+            <h4>Note on Email & WhatsApp</h4>
+            If you wish to use your own WhatsApp Business Account, or your own email address when using the `email`
```
Comment (SMadani) [inline definitions/verify.v2.yml:27]: Probably don't need the comma on this line but otherwise LGTM.

**Review by SMadani — APPROVED** [review]

File: definitions/verify.v2.yml (hunk 1 above)
Comment (superchilled) [inline definitions/verify.v2.yml:27]: The sentence construction seems a little unclear to me. Perhaps something like:

```suggestion
            If you wish to use your own WhatsApp Business Account for the WhatsApp channel, or your own email address when using the `email` channel, when populating the  in the 'from' field, please note the following:
```

File: definitions/verify.v2.yml [hunk 2]
```diff
@@ -12,18 +12,26 @@ servers:
...
+            channel in the 'from' field, please note the following:
+              <ul>
+                <li><strong>&nbsp;&nbsp;• Testing for these channels is self-service.</strong></li>
```
Comment (superchilled) [inline definitions/verify.v2.yml:30]: Will customers reading this know what we mean by self-service in this context?

**Review by superchilled — COMMENTED** [review]
Looks good overall, just a suggestion re the wording for the call-out, and a couple of questions.

Also wondering if we maybe want to link out to some documentation in the call-out? (if we even have documentation on this? If not, do we want to add some?)

**Pull request merged**